docs/wiki/EL/precompiled-contracts.md

```diff
@@ -0,0 +1,56 @@
+# Precompiled contracts
+
+Precompiled contracts are a set of special accounts, each containing a built-in function with a determined gas cost, often related to cryptography. Currently, they are defined at addresses ranging from `0x01` to `0x0a`. [EIPs](https://eips.ethereum.org/) can introduce new precompiles as part of hard forks.
+
+Unlike contract accounts, precompiles are part of the Ethereum protocol and implemented by execution clients. This presents an intersting caveat - their `EXTCODESIZE` within EVM is 0. Nonetheless, they function like contract accounts with code when executed.
```

Comment:
Maybe quickly mention the relation to EVM in the intro, that precompiles are basically bundled with evm and remove overhead

Also typo in 'intersting'

Reply:
I have added the section "Precompiles vs. opcodes" to expand on this.